publiq-ui/pub-modal.vue

```diff
@@ -0,0 +1,80 @@
+<template>
+  <b-modal
+    cancel-variant="outline-secondary"
+    :title="title"
+    :ok-title="confirmTitle"
+    :cancel-title="cancelTitle"
+    title-class="title"
+    modal-class="modal"
+    content-class="content"
+    body-class="body"
+    :visible="visible"
+    :scrollable="false"
+    :hide-footer="isContentVariant"
+    :hide-header="isQuestionVariant"
+    :size="isContentVariant ? 'xl' : 'md'"
+    v-on="$listeners"
```

Comment:
We should probably follow the same principle for listeners as for props: Only expose/pass those that we actually use.

Reply:
I was wondering though, with listeners, what is the harm in exposing them all? If we expose the ones we want it is true in storybook they'll be documented better

Reply:
Yeah I was thinking mostly in having a definite list of props/listeners we have "support" so we don't have to go look at what listeners are registered on each usage of a component if we ever move away from Bootstrap. Maybe it's a bit overkill though. To be honest I didn't even think of this difference with props yet because in React event listeners are also just props (at least in functional components). So I (incorrectly assumed) we were also in agreement about event listeners when we said we would only allow specific props we actually need.

Reply:
It would be nice to see the events in Storybook though. E.g. when you open a modal, close a modal, click a button, etc. You could just play around with it and see what events are being emitted instead of having to go look for them in the Bootstrap docs.

Reply:
@geoffreydhuyvetters @bertramakers It is ok for me if we also only emit specific events. it will be better documented in storybook. We'll only have a bit more work maintaining them (maybe). But when potentially switching away from bootstrap it will be clearer which events we use and need. Ok for everyone? If so @bertramakers can add this rule to the documentation

Reply:
I think @geoffreydhuyvetters is actually working on another project today, so if we both agree I propose we do it like this and document it indeed, and we can still discuss it next week if he sees any problems with it we might have missed.